.github/docker_files/docker_file_ci_ubuntu_22_04/DockerFile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:focal
```

Comment:
maybe 24.04 right away? Not sure if you communicated with @pooyan-dadvand 
EDIT I think this is unused, I never finished it :see_no_evil: